first_responder.md

```diff
@@ -0,0 +1,121 @@
+
+### Proactively Check for Production Problems
+
+See "Proactively Check for Production Problems" section below for specifics. 
+
+### Triage Production Problems
+
+If a user reports a problem, or if one is surfaced from monitoring, the first responder is meant to timebox an investigation of the problem (_TBD: 30 min?_).  See "Triage Production Problems" section for more specifics.
+
+### Improve Troubleshooting Documentation when Needed
+
+If you need to triage or troubleshoot a problem and realize some documentation is missing, please provide it.  List of appropriate places:
+* https://github.com/sul-dlss/DevOpsDocs - e.g. what an ops or devops person would need to know to handle the situation
+* README or other top level markdown doc in codebase (viewable via github)
+  * Which codebase would need to be apparent from the problem
+* Wiki for the codebase
+* ? - in general, just try to consider where the person interested in the info might look first.  A user might go to the wiki, a dev might go to the README, ops folks might head to DevOpsDocs.  Use your best judgement and ask for feedback if unsure.
+
+The above documents should be useful and current.  Please submit improvements as PRs for review.
+
+If for some reason documentation is a significant undertaking, the call for documentation can be filed as an issue and prioritized/resourced by management.
+
+### Improve First Responder Instructions as Needed
+
+We need this document to be useful and current.  Please submit improvements as PRs for  review.
+
+## What if first responder isn't available?
+
+The idea is for the first responder to be "interruptible" for production problems during his/her week of coverage.  If there are significant blocks of time when this isn't true (e.g. ½ day meeting with no access to slack or email), or if life happens (illness, family emergency, etc.), hopefully the first responder can arrange for coverage.  ("I'll take one of your days if you can cover Tues for me").  Please notify `#dlss-infrastructure` Slack channel of changes.
+
+When the first responder can't arrange for coverage, the default would be for the "on deck" responder to become first responder.  The "on deck" person is the first responder for the following week.
+
+## Proactively Checking for Production Problems
+
+At the very least, the first responder should be watching incoming alerts from Honeybadger.
+
+Other places to monitor:
+* Nagios alerts
+* feedback email lists, e.g. argo-feedback, sinopia-feedback, etc
+* the `#dlss-infrastructure` channel
+* Slack channels relevant to applications in the portfolio, e.g. `#pre-assembly`
+* resque dashboards?  e.g. for preservation catalog?
+* cron daemon emails
+* Github security alerts
+* If you hear about possible security issues through other avenues (e.g. `#iso-public` on Slack, your consumption of the news in general), and you feel that the issues may be relevant and uncaptured, file an issue and call attention to it as needed.
+
```

Comment:
other tips for places to watch are welcome (@ndushay seemed esp interested in getting @jcoyne and/or @justinlittman to add anything they know of that might be missing)